client/src/components/details/OverviewModule.jsx

```diff
@@ -1,34 +1,46 @@
 import React from 'react';
 import StyleLib from '../../assets/Stylesheet.jsx';
 import DetailsLib from './assets/Stylesheet.jsx';
-import api from '../../../../client/src/lib/api.js';
+import api from '../../lib/api.js';
 
 import Header from './header.jsx';
 import ProductInformation from './productInformation.jsx';
 import Image from './image.jsx';
+import ExpandedImage from './expandedImage.jsx';
 import Description from './description.jsx';
+import ProductInfoObject from '../../lib/productInfoObjectTemplate.js';
 
 const OverviewModule = () => {
-  const [styleList, setStyleList] = React.useState([
-    /* ARRAY OF STYLES */
-    /*filler data (not used):*/
-    0, 1, 2, 3, 4, 5, 6
-  ]);
+  const [product, setProduct] = React.useState(ProductInfoObject);
+  const [styleList, setStyleList] = React.useState(ProductInfoObject.styles.results);
+  const [favorite, setFavorite] = React.useState(false);
+  const [openModal, setOpenModal] = React.useState(false);
+
+  React.useEffect(() => refreshOverviewModule(40399), []);
+
+  const refreshOverviewModule = (p_id) => {
+    api.collectProductInfo(p_id)
+      .then(prod => {
+        // console.log(prod)
+        setProduct(prod);
+        setStyleList(prod.styles.results)
+        return;
+      })
+      .catch(err => console.log(err));
+  }
 
   return (
     <div>
+      {openModal ? <ExpandedImage openModal={openModal} setOpenModal={setOpenModal}/> : <></>}
```

Comment:
Super minor cleanup, but something that seems to be a common pattern for conditional rendering is:
`{openModal && <ExpandedImage openModal={openModal} setOpenModal={setOpenModal}/>}`
It's equivalent to what you have above but gets rid of the ?, the :, and the extra <></> at the end. 

However, in this case, you don't even need that! Unless I'm missing something, you should be able to just use:
`<ExpandedImage openModal={openModal} setOpenModal={setOpenModal}/> `
here, since the modal takes care of rendering or not rendering itself based on the isOpen prop.